ui/raidboss/data/06-ew/trial/golbez-ex.txt

```diff
@@ -1,6 +1,394 @@
 ### The Voidcast Dais (Extreme)
+# -ii 84B5 84B7 849E 84A8 84AA 8481 86B2
 
 hideall "--Reset--"
 hideall "--sync--"
 
 0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
+
+0.0 "--sync--" sync /Engage!/ window 0,1
+# Exclude the first auto since it's delayed wildly depending on tank movement
+9.4 "--sync--" sync / 14:[^:]*:Golbez:8464:/ window 15,15
+12.4 "Terrastorm (cast)" sync / 1[56]:[^:]*:Golbez:8464:/
+21.5 "Terrastorm (hit)" sync / 1[56]:[^:]*:Golbez:8466:/
+22.5 "Lingering Spark (snapshot)" sync / 1[56]:[^:]*:Golbez:8468:/
+26.6 "Lingering Spark (explode)" sync / 1[56]:[^:]*:Golbez:846A:/
+31.8 "Phases of the Blade (cast)" sync / 1[56]:[^:]*:Golbez:86DB:/
+33.6 "Phases of the Blade (front)" sync / 1[56]:[^:]*:Golbez:86DD:/
+35.3 "Phases of the Blade (back)" sync / 1[56]:[^:]*:Golbez:86F2:/
+44.6 "Binding Cold" sync / 1[56]:[^:]*:Golbez:84B3:/
+
+52.8 "--jump--" sync / 1[56]:[^:]*:Golbez:84B8:/
+57.9 "Gale Sphere" sync / 1[56]:[^:]*:Golbez:844D:/
+68.6 "--sync--" sync / 1[56]:[^:]*:Golbez:844E:/
+71.2 "Gale Sphere 1" sync / 1[56]:[^:]*:Gale Sphere:845[89AB]:/
+73.6 "Arctic Assault" sync / 1[56]:[^:]*:Golbez:845F:/
+74.6 "Gale Sphere 2" sync / 1[56]:[^:]*:Gale Sphere:845[89AB]:/
+74.6 "Arctic Assault" sync / 1[56]:[^:]*:Golbez:8461:/
```

Comment:
Can you `-ii 8461` and remove all those lines? Mostly there's only one Arctic Assault going on here, so usually I just leave the boss ability and ignore the damage (unless it's a weird timing snapshot).